### ISSUE TITLE
Is the bleu score of EN-DE BERT-fused NMT model reported in paper's Table 3 is case-sensitive or case-insensitive?

### ISSUE THREAD

**Issue by nomadlx** [bert-nmt/bert-nmt#22]
I only find a little info about case in Appendix.1.

> for En-De, we lowercase all words, split 7k sentence pairs from the training dataset for validation and concatenate dev2010, dev2012, tst2010, tst2011, tst2012 as the test set.

Is this mean the bleu score is case-insensitive？

**Comment by teslacool** [comment]
Yes, following [Fairseq](https://github.com/bert-nmt/bert-nmt/blob/master/examples/translation/prepare-iwslt14.sh), we lowercase all the samples in IWSLT'14 En<->De and thus the bleu score is case-insensitive.

**Comment by nomadlx** [comment]
Thank you. Another thing… Can you provide the fairseq commit version that your code base on?     It seem that your code can’t support to train a origin transformer model.

**Comment by teslacool** [comment]
a8f28ecb63ee01c33ea9f6986102136743d47ec2
